templates/tfengine/components/audit/main.tf

```diff
@@ -44,7 +44,7 @@ module "bigquery_export" {
   source  = "terraform-google-modules/log-export/google"
   version = "~> 5.0.0"
 
-  log_sink_name          = "bigquery-audit-logs-sink"
+  log_sink_name          = "{{printf "%sbigquery-audit-logs-sink" (get . "logs_sink_prefix" "")}}"
```

Comment:
I think a more flexible approach would be to simply support bigquery_sink_name and storage_sink_name fields that hte user can set to whatever they like.

Reply:
Done